backend/src/packages/chaiNNer_standard/image/create_images/text_as_image.py

```diff
@@ -0,0 +1,206 @@
+    },
+    TextAsImagePosition.BOTTOM_LEFT: {
+        "x": np.array([0, 0.5]),
+        "y": np.array([1, -0.5]),
+    },
+    TextAsImagePosition.BOTTOM_CENTERED: {
+        "x": np.array([0.5, 0]),
+        "y": np.array([1, -0.5]),
+    },
+    TextAsImagePosition.BOTTOM_RIGHT: {
+        "x": np.array([1, -0.5]),
+        "y": np.array([1, -0.5]),
+    },
+}
+
+
+@create_images_group.register(
+    schema_id="chainner:image:text_as_image",
+    name="Text As Image",
+    description="Create an image using any text.",
+    icon="MdTextFields",
+    inputs=[
+        TextInput("Text", multiline=True),
+        BoolInput("Bold", default=False),
+        BoolInput("Italic", default=False),
+        ColorInput(channels=[3], default=Color.bgr((0, 0, 0))),
+        EnumInput(
+            TextAsImageAlignment,
+            label="Text alignment",
+            option_labels=TEXT_AS_IMAGE_ALIGNMENT_LABELS,
+            default=TextAsImageAlignment.CENTERED,
+        ),
+        NumberInput(
+            "Width",
+            minimum=1,
+            maximum=None,
+            controls_step=1,
+            precision=0,
+            default=500,
+        ),
+        NumberInput(
+            "Height",
+            minimum=1,
+            maximum=None,
+            controls_step=1,
+            precision=0,
+            default=100,
+        ),
+        EnumInput(
+            TextAsImagePosition,
+            label="Position",
+            option_labels=TEXT_AS_IMAGE_POSITION_LABELS,
+            default=TextAsImagePosition.CENTERED,
+        ),
+    ],
+    outputs=[
+        ImageOutput(
+            image_type="""
+                Image {
+                    width: Input5,
+                    height: Input6,
+                    channels: 4
+                }
+                """,
+            assume_normalized=True,
+        )
+    ],
+)
+def text_as_image_node(
+    text: str,
+    bold: bool,
+    italic: bool,
+    color: Color,
+    alignment: TextAsImageAlignment,
+    width: int,
+    height: int,
+    position: TextAsImagePosition,
+) -> np.ndarray:
+    path = TEXT_AS_IMAGE_FONT_PATH[int(bold)][int(italic)]
+    font_path = os.path.join(
+        os.path.dirname(sys.modules["__main__"].__file__),  # type: ignore
+        f"fonts/{path}",  # type: ignore
+    )
+
+    lines = text.split("\n")
+    line_count, max_line = len(lines), max(lines, key=len)
+
+    # Use a text as reference to get max size
+    font = ImageFont.truetype(font_path, size=100)
+    w_ref, h_ref = font.getsize(max_line)[0], font.getsize("[§]")[1]
+
+    # Calculate font size to fill the specified image size
+    w = int(width * 100.0 / w_ref)
+    h = int(height * 100.0 / (h_ref * line_count))
+    font_size = min(w, h)
+    font = ImageFont.truetype(font_path, size=font_size)
+    w_text, h_text = font.getsize(max_line)
+    h_text *= line_count
+
+    # Text color
+    ink = 255 * np.array(color.value)
+    ink = tuple(ink.astype("uint8"))
```

Comment:
Please use `to_uint8` from `nodes.impl.image_utils`.

```suggestion
    ink = tuple(to_uint8(color.value))
```